test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/NpgsqlCommandTests.cs

```diff
@@ -39,17 +39,22 @@ public void SubmitsTracesWithNetStandard(string packageVersion, bool enableCallT
             // 2 interfaces: IDbCommand and IDbCommand-netstandard).
             // Once this is fully supported, this will add another 2 complete groups instead.
 #if NET452
-            var expectedSpanCount = 50; // 7 queries * 7 groups + 1 internal query
-#else
             var expectedSpanCount = 78; // 7 queries * 11 groups + 1 internal query
+#elif NET461
+            var expectedSpanCount = 134; // 7 queries * 11 groups + 1 internal query
+#else
+            // Update to final value once tests pass
+            var expectedSpanCount = 135; // 7 queries * 11 groups + 1 internal query
```

Comment:
As above RE comments

Reply:
I put off updating these 😅 Done in f9829db87629aa5c357cc4ad5a69a51fcd1da1ad